django/forms/widgets.py

```diff
@@ -77,12 +77,24 @@ def render_css(self):
             ) for path in self._css[medium]
         ] for medium in media])
 
-    def absolute_path(self, path, prefix=None):
+    def absolute_path(self, path):
+        """
+        Return absolute path to static asset.
+
+        Both absolute and relative paths are accepted.
+        An absolute path will be returned unchanged, where an relative
+        path will be transformed into the staticfile storage absolute path.
+
+        Args:
+            path (str): Relative or absolute path to static asset.
+
+        Returns:
```

Comment:
It would be helpful if django would commit to pep257 and numpy or google docstring format to shorten the review cycle.